Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -360,6 +374,10 @@ public class Event: NosManagedObject, VerifiableEvent {
                 " OR $reference.marker = 'reply'" +
                 " OR $reference.marker = nil" +
                 ").@count = 0"
+            ),
+            NSPredicate(
+                format: "identifier <> %@",
```

Comment:
I'm not familiar with `<>` in a predicate. Is this the same as `!=` or does it mean something else?

Reply:
They are the same. `<>` comes from the SQL world.

Reply:
In that case, could we use `!=`? That would be consistent with other predicates -- it looks to me like this would introduce our first use of the `<>` operator, and I'd love to stick with what we already use. I had to look up `<>` and even then didn't find an answer, which is why I asked here. I wouldn't want others to look at this, not know what it means, try to look it up, not find an answer, and have to ask us.

Reply:
I agree with you, we should be using `!=` to be consistent. It's also true that it's hard to find a documentation for `<>`, I had to go deep in Apple's documentation and found it here: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Predicates/Articles/pSyntax.html#//apple_ref/doc/uid/TP40001795-215832 under Basic Comparisons. It was last updated in 2014 xd.